src/main/python/fine_tuning/xfold.py

```diff
@@ -18,7 +18,8 @@
 import logging
 import json
 
-logging.basicConfig()
```

Comment:
You added `logging.basicConfig()` for other files but remove it from this file. Any particular reason?

Reply:
No idea why it was removed.
Reverted